source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py

```diff
@@ -988,12 +988,7 @@ def transform_points(
 
 @torch.jit.script
 def unproject_depth(depth: torch.Tensor, intrinsics: torch.Tensor) -> torch.Tensor:
```

Comment:
Although it mentions in the doc string that depth here should be orthogonal maybe good to also mention it in the method name like `unprojec_orthogonal_depth` as I feel like the doc string comment could be missed

Reply:
I added a parameter to the function which user can set as True/False for orthogonal/perspective depth image respectively.